backend/app/schemas.py

```
@@ -1 +1,99 @@
+    execution_time: float = Field(..., description="実行時間（秒）")
+
+
+class TableSchemasResponse(BaseModel):
+    """テーブルスキーマ情報レスポンス"""
+    tables: List[Dict[str, Any]] = Field(
+        ..., description="テーブルスキーマ情報"
+    )
+    total_count: int = Field(..., description="テーブル総数")
+
+
+class HealthResponse(BaseModel):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)